src/views/collection/Seo/index.tsx

```diff
@@ -31,6 +32,8 @@ function Seo({
     canonical: `/${slug}/`,
   })
 
+  useAddHead({ title: collectionSeo.title })
```

Comment:
Why is it collectionSeo.title and not metadata.title?

Reply:
You're right, i was looking the useMetadata code, and it return ```tsx { ..., title, ...}```

Reply:
ooops, I'm a little bit late to the party. We use `collctionSeo.title` because. for each collection, the user registers an specific title for that page on our admin